gulpfile.js

```diff
@@ -48,7 +48,7 @@ gulp.task('sass', function () {
     .pipe(stripCssComments({ preserve: false }))
     .pipe(postcss([postcssPrefix({
       prefix: bootstrapNamespace + ' ',
-      exclude: [/^body/, /page-civicrm-case/, outsideNamespaceRegExp]
+      exclude: [/^body/, /contact-popover-container/, /page-civicrm-case/, outsideNamespaceRegExp]
```

Comment:
We don't need to add `contact-popove-container` inside gulpfile.js. If you will take a look in the gulpfile there is a selector `.__outside-namespace` which when added a scaffolding to a selector, the css inside it doesn't get `#bootstrap-theme` prefix.

also children classes don't need to wrapped inside parent selector. Since they are independent and unique classes they can be used independently.
So overall the structure of `_contact-popover.scss` should be.

```
.___outside-namespace {
  .contact-popover-container {
    color: $gray-darker;
    max-width: 90vw;
    padding: 20px;
    width: $contact-popover-width;

    .popover-content {
      padding: 0;
    }
  }
  .civicase__contact-popover__header {
    h2 {
      line-height: 24px;
      margin: 0;
    }

    // HRs outside of #bootstrap-theme don't have margins
    hr {
      margin: 16px 0;
    }
  }

  .civicase__contact-popover__column {
    float: left;
    width: 46%;
  }

  .civicase__contact-popover__column + .civicase__contact-popover__column {
    width: 54%;
  }

  // floating is needed to give some margin between groups
  .civicase__contact-popover__detail-group {
    float: left;
    margin-bottom: 10px;
    width: 100%;
  }

  .civicase__contact-popover__detail-header,
  .civicase__contact-popover__detail-value {
    color: $gray-dark;
    float: left;
    line-height: 18px;
    overflow-x: hidden;
    text-overflow: ellipsis;
    white-space: nowrap;
    width: 55%;

    strong {
      color: $gray-darker;
      font-weight: 600;
    }
  }

  .civicase__contact-popover__detail-header {
    clear: both;
    width: 45%;
  }  
}
```